tests/c/CMakeLists.txt

```diff
@@ -1,56 +1,50 @@
-cmake_minimum_required(VERSION 3.5)
-
-find_package(GTest)
-
-# FIXME currently requires static lib, this should include build static csound condition
-if(BUILD_TESTS AND BUILD_STATIC_LIBRARY AND GTest_FOUND)
-    add_executable(unittests
-        channel_tests.cpp
-        cscore_tests.cpp
-        csound_circular_buffer_test.cpp
-        csound_data_structures_test.cpp
-        csound_debugger_test.cpp
-        csound_message_buffer_test.cpp
-        csound_orc_compile_test.cpp
-        csound_orc_semantics_test.cpp
-        csound_type_system_test.cpp
-        engine_test.cpp
-        io_test.cpp
-        server_test.cpp
-    )
+add_executable(unittests
```

Comment:
I just removed the condition at the start didn't change much else in this file. Git just isn't lining up the lines correctly